### PR TITLE
Draft minutes for Packaging WG Meeting of 20/09/18

### PR DESCRIPTION
**Pull request by drbenmorgan**
Composed from Live Notes, thanks to @graeme-a-stewart for updates!

### CODE REVIEW

**Comment by drbenmorgan** [comment]
Checking the Travis results, it looks like these are the usual HTMLProofer failures. Certainly not in the file this PR touches.

File: organization/_posts/2018/2018-09-19-packaging.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+- FCC Use Case:
+  - Use LCG CVMFS install (base), plus Spack "packages.yaml" describing them as “system” packages
+    - Graeme: is there a script to generate this? Yes.
+  - Allows override of LCG packages as required
+  - Layered config: FCC specific packages used first, then LCG then Spack base
+- Workflow:
+  - From CVMFS, merge LCG and FCC packages.yaml
+  - Run `spack install <metapackage>` using this, outputting binary tarballs
+  - Binary tarballs deployed/published on CVMFS, plus spack view creation
+  - Graeme: Have looked at combining Build/Publication node with Tomas Stefan (summer student), presentation is [here](https://indico.cern.ch/event/746546/contributions/3086271/attachments/1708942/2754445/Deployment_of_ATLAS_releases.pdf).
```
Comment (graeme-a-stewart) [inline organization/_posts/2018/2018-09-19-packaging.md]: Small point, but maybe rephrase this to

"Graeme: We have tested parallel publication to CVMFS from many nodes that not only performs better, but simplifies the workflow as the build node can also take care of publishing the results (see [presentation](https://indico.cern.ch/event/746546/contributions/3086271/attachments/1708942/2754445/Deployment_of_ATLAS_releases.pdf) from Tomas Stefan's summer student project)."

File: organization/_posts/2018/2018-09-19-packaging.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+- Ben C: Agree on Containers+PMs being friends, have seen issues with trying to run modern code natively on old (in this case SLC5) grid nodes.
+- Todd: Would like to get glibc dependency in Spack at some point.
+- Hadrien: One other use case, somewhat specific, for containers is when using a rolling Linux distro. When rolling updates compiler etc, spack etc may want/need to update/rebuild everything.
+
+AOB
+===
+- Next meeting 3 October: [https://indico.cern.ch/event/758817/](https://indico.cern.ch/event/758817/)
+  - One presentation from FNAL on Spack/Spack-dev
+  - As ever, more contributions welcome, contact Graeme and Ben
+
```
Comment (graeme-a-stewart) [inline organization/_posts/2018/2018-09-19-packaging.md]: We could perhaps all add here:

- Todd: very happy with collaboration with HEP so far, would like to prioritise important HEP use cases (very good PRs received)
  - Graeme and Todd will keep in touch to better communicate and align goals and developments

**Comment by graeme-a-stewart** [comment]
Thanks as ever!
